pages/article/[id].js

```diff
@@ -104,6 +104,11 @@ const useStyles = makeStyles(theme => ({
     width: '100%',
     borderRadius: theme.shape.borderRadius,
   },
+  attachmentImage: {
+    width: '100%',
+    maxWidth: 800,
```

Comment:
Seems that 800 is a bit too high, vertical images can hit max-height first XD

![image](https://user-images.githubusercontent.com/108608/168965648-bc9041ef-529c-4f9b-a11d-1f71c904094d.png)

Suggest we `max-width` can be around 400, and we can remove `max-height` so that image height can be completely determined by image's apsect ratio.